fixed_frame: "aoa"
estimate_frame: "twr_position"
mode: "RSSI"

ids: [
  "an1",
  "an2",
  "an3",
  "an4",
]

positions: [
  #x,y,z, qx, qy, qz, qw
   0.065,  0.0, 0.0, 0.0, 0.0,  0.0,       1.0,
   0.0,   -0.065, 0.0, 0.0, 0.0, -0.7071068, 0.7071068,
  -0.065,  0.0, 0.0, 0.0, 0.0,  1.0,       0.0,
   0.0,    0.065, 0.0, 0.0, 0.0,  0.7071068, 0.7071068,
]

frames: [
  "aoa1",
  "aoa2",
  "aoa3",
  "aoa4",
]

